packages/abap-deploy-config-inquirer/src/prompts/conditions.ts

```diff
@@ -144,12 +144,11 @@ export function showPasswordQuestion(): boolean {
 /**
  * Determines if the UI5 app deploy config question should be shown (UI5 Abap Repo name & Description).
  *
- * @param ui5AbapPromptOptions - UI5 Abap Repo prompt options
+ * @param appType - application type
  * @returns boolean
  */
-export function showUi5AppDeployConfigQuestion(ui5AbapPromptOptions?: UI5AbapRepoPromptOptions): boolean {
-    // if hideIfOnPremise option is true and the target system is on-premise, hide the prompt
-    if (!ui5AbapPromptOptions?.hide && ui5AbapPromptOptions?.hideIfOnPremise && !PromptState.abapDeployConfig?.scp) {
+export function showUi5AppDeployConfigQuestion(appType?: AppType): boolean {
+    if (PromptState.abapDeployConfig?.scp || (appType === 'Fiori Adaptation' && !PromptState.abapDeployConfig.isS4HC)) {
```

Comment:
Why has `hide` & `hideIfOnPremise` options been removed? This prompt would now be hidden for all systems which are `scp` which would be a major change for us.